src/server/utils/processDocInfos.ts

```diff
@@ -25,6 +26,12 @@ export function processDocInfos(
         );
       }
       const route = url.substr(baseUrl.length);
+
+      // ignore files
+      if (ignoreFiles?.some((reg: RegExp | string) => route.match(reg))) {
```

Comment:
I prefer to consider strings as glob match patterns. It's ok to support RegExp only first, but we should not accept strings as implicit RegExps.

Reply:
Yes, that's right, i have fix this problem~